docs/synthetics/status-pages.md

```diff
@@ -0,0 +1,37 @@
+title: Status Pages
```

Comment:
Should we have this under docs/synthetics if the SPs will be more general?  I suppose we could leave it under synthetics for now and redirect after we move this.

Reply:
Yes, we can leave it under Synthetics for now since currently it is part of Synthetics.